### DISCUSSION TITLE
zigbee2mqtt send to iobroker and home-assistant

### DISCUSSION THREAD

**Discussion by Sn4k3E4t3r** [Koenkk/zigbee2mqtt#14286] [General]
Hi. I have two Smart Home Systems running. (iobroker is for me the better System but have no mobile app) For simple things the Home Assistant App is there.

What is the best Way to send zigbee devices to both Systems? I have read that zigbee2mqtt only can handle one Sever

So perhabs somebody have an Idea. Perhabs over an MQTT Broker Enviroment

**Comment by DubhAd** [comment]
Point both ioBroker and Home Assistant at the same MQTT broker

**Reply by Sn4k3E4t3r** [reply]
what do you mean exactly? Did you have some documentation?

**Reply by DubhAd** [reply]
Just that.

Install Mosquitto (or your choice of broker) somewhere.

Connect Zigbee2MQTT to that broker - ensure you've also enabled the [Home Assistant integration](https://www.zigbee2mqtt.io/guide/configuration/homeassistant.html).

Connect ioBroker to that broker, as a client.

Connect Home Assistant to that broker.

Now both ioBroker and HA will see all the MQTT messages.
